_build/templates/default/sass/login.scss

```diff
@@ -217,6 +217,17 @@ input[type="checkbox"] {
   }
 }
 
+
+#modx-login-btn {
+  border: 1px solid transparent;
+  &:hover {
```

Comment:
Maybe worth turning this into `&:hover, &:focus {` to also cover keyboard users? Or is there already an outline shown for those?